packages/hash/frontend/src/pages/[account-slug]/entities/[entity-uuid].page/shared/use-create-new-entity-and-redirect/get-default-value-of-property-type.ts

```diff
@@ -0,0 +1,38 @@
+import { PropertyType } from "@blockprotocol/type-system-web";
+import { Subgraph } from "@hashintel/hash-subgraph";
+import {
+  isPropertyValueArray,
+  isPropertyValueNested,
+} from "../../../../../../lib/typeguards";
+import { generateDefaultProperties } from "./generate-default-properties";
+
+/**
+ * @todo this will be deleted when https://app.asana.com/0/1203312852763953/1203433085114587/f (internal) is implemented
+ */
+export const getDefaultValueOfPropertyType = (
```

Comment:
This really doesn't need to be its own file. I know this seems nit picky, but there is a weight to separating things, and this is only being used by `generate-default-properties`, and both are temporary, and neither file is that long. I'd consider combining them.